ontimize/components/table/extensions/pipes/o-table-row-class.pipe.ts

```diff
@@ -0,0 +1,10 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({ name: 'oTableRowClass' })
+export class OTableRowClassPipe implements PipeTransform {
+
+  transform(row: object, rowClassFn?: (row: object) => string | string[]): string | string[] {
```

Comment:
It would be great if function also receives as argument row status as selected, hover, etc. or row index

Reply:
It is possible to include the row index as a parameter.
The pipe transformation is executed every time the parameter `row` changes, not when the change detection is executed, so it would receive always the same values

Reply:
ok